app/views/api_new/subtransaction_payments/_subtransaction_payment.json.jbuilder

```diff
@@ -7,6 +7,8 @@ json.type 'payment'
 json.id subtransaction_payment.paymentable.houid
 json.created subtransaction_payment.created.to_i
 
+json.legacy_id subtransaction_payment.legacy_payment_id
```

Comment:
Can you please define what a 'subtransaction_payment'?

Reply:
A subtransaction payment is our modern version of "payment" as we show it in the new api. It corresponds to a charge, refund, dispute, dispute reversal or a offline transaction payment created. Each subtransaction payment will have a legacy_payment which is of type `Payment`